app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -152,46 +173,62 @@ class ReadingTextSizeFragmentTest {
     )
   }
 
-  private fun clickSeekBar(position: Int): ViewAction {
-    return GeneralClickAction(
-      Tap.SINGLE,
-      CoordinatesProvider { view ->
-        val seekBar = view as SeekBar
-        val screenPos = IntArray(2)
-        seekBar.getLocationInWindow(screenPos)
-        val trueWith = seekBar.width - seekBar.paddingLeft - seekBar.paddingRight
-
-        val percentagePos = (position.toFloat() / seekBar.max)
-        val screenX = trueWith * percentagePos + screenPos[0] + seekBar.paddingLeft
-        val screenY = seekBar.height / 2f + screenPos[1]
-        val coordinates = FloatArray(2)
-        coordinates[0] = screenX
-        coordinates[1] = screenY
-        coordinates
-      },
-      Press.FINGER, /* inputDevice= */ 0, /* deviceState= */ 0
-    )
-  }
-
-  private fun seekBarProgress(progress: Int): TypeSafeMatcher<View> {
+  /** Matcher for comparing the textSize of content inside a TextView to the expected size. */
+  private fun matchTextViewTextSize(expectedSize: Float): TypeSafeMatcher<View> {
     return object : TypeSafeMatcher<View>() {
       override fun describeTo(description: Description?) {
-        description?.appendText("SeekBarProgress")
+        description?.appendText("TextView with text size: $expectedSize")
       }
 
-      override fun matchesSafely(item: View?): Boolean {
-        return (item as SeekBar).progress == progress
+      override fun matchesSafely(item: View): Boolean {
+        (item as TextView).apply {
+          val pixels = textSize
+          val actualSize = pixels / resources.displayMetrics.scaledDensity
+          return expectedSize.compareTo(actualSize) == 0
+        }
```

Comment:
This is working for me. Also, try to put the wrong values of `matchTextViewTextSize(expectedSize)` and see if the test fails.  

```suggestion
        return item is TextView && item.textSize == expectedSize
```

Reply:
@ARJUPTA conversation should only be resolved by the one who created as it helps in PR review.

Reply:
Pardon @anandwana001, the comment got resolved automatically as I directly commited your changes from the Github (ie. commit suggestion button)

Reply:
ah, I didn't know about this GitHub feature. No worries.